packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -470,7 +470,9 @@ export default class RestApi {
     api.get(
       '/:collection(vdis|vdi-snapshots)/:object.:format(vhd|raw)',
       wrap(async (req, res) => {
-        const stream = await req.xapiObject.$exportContent({ format: req.params.format })
+        const preferNbd = req.query.preferNbd === 'true'
+        const nbdConcurrency = req.query.nbdConcurrency && parseInt(req.query.nbdConcurrency)
```

Comment:
[Avoid using `parseInt()`](https://dev.to/darkmavis1980/you-should-stop-using-parseint-nbf), use `+` or `Number()` instead.

Also, `ifDef` can be useful here:

```suggestion
        const nbdConcurrency = ifDef(req.query.nbdConcurrency, Number)
```

Reply:
I really want an integer here

Reply:
This is what `+` or `Number` does :slightly_smiling_face:

Reply:
+'1.2'  gives me 1.2
Number('1.2') also gives me 1.2

I really want to have 1 , or at least an error .

Reply:
I'm not fond of it, but no big deal in this case.